.travis.yml

```diff
@@ -3,26 +3,22 @@ sudo: false
 
 branches:
   only:
-    - master
+    #- master
+    - conda_install
```

Comment:
Um, does this belong? It looks like you're telling travis to test only on the conda-install branch?